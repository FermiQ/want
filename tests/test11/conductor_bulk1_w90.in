
&INPUT_CONDUCTOR
   postfix = '_bulk1_w90'
   dimC = 6
   calculation_type = 'bulk'
   ne   = 1000
   emin = -10.0
   emax =   5.0
   datafile_C = "./SCRATCH/wannier90_Pt1.chk"
   transport_dir = 3
   shift_R =  6.7
   shift_L =  6.7
   shift_C =  6.7
/

<HAMILTONIAN_DATA>

   <H00_C rows="ALL" cols="ALL"  />
   <H_CR  rows="ALL" cols="ALL"  />

</HAMILTONIAN_DATA>

