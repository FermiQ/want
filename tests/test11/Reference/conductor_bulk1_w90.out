
  ======================================================================
              =                                            =
              =     *** WanT *** Wannier Transport Code    =
              =        (www.wannier-transport.org)         =
              =      Ultra Soft Pseudopotential Implem.    =
              =                                            =
  ======================================================================


  Program <conductor>  v. 2.3.0  starts ...
  Date  6Apr2010 at  7:54: 5 

     Parallel run, # proc:    6

          BUILT :    Tue Apr  6 07:04:52 2010
           HOST :    x86_64-unknown-linux-gnu
           ARCH :    x86_64
             CC :    icc
            CPP :    cpp
            F90 :    mpif90
            F77 :    ifort
         DFLAGS :    -D__INTEL -D__FFTW -D__MPI -D__PARA
      BLAS LIBS :    -L/home/ferretti/want/install -lmkl_em64t_nothread
    LAPACK LIBS :    
       FFT LIBS :    
      MASS LIBS :    



  ======================================================================
  =  Conductor Init                                                    =
  ======================================================================

  ./SCRATCH/wannier90_Pt1.chk converted to internal fmt

  ======================================================================
  =  INPUT Summary                                                     =
  ======================================================================

  <INPUT>
          Calculation Type :     bulk
                    prefix :     
                   postfix :     _bulk1_w90
                  work_dir :     ./
               L-lead dim. :         6
            conductor dim. :         6
               R-lead dim. :         6
       Conductance Formula :     landauer
       Transport Direction :         3
          Have Correlation :     FALSE
              Write k-data :     FALSE
       Max iteration number:       200

                    nprint :        20

        Conductor datafile :     ./SCRATCH/wannier90_Pt1.chk.ham
  </INPUT>


  <ENERGY_GRID>
                 Dimension :       1000
                Min Energy :      -10.00000
                Max Energy :        5.00000
               Energy Step :        0.01502
                     Delta :        0.00001
             Smearing Type :     lorentzian
             Smearing grid :      20000
             Smearing gmax :       25.00000
                   Shift_L :        6.70000
                   Shift_C :        6.70000
                   Shift_R :        6.70000
                Shift_corr :        0.00000
  </ENERGY_GRID>


  <K-POINTS>
       nkpts_par =    1
       nrtot_par =    1
        use_symm = TRUE

       Parallel kpoints grid:        nk = (  1  1  1 )   s = (  0  0  0 )
       k (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000

       Parallel R vector grid:       nr = (  1  1  1 )
       R (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000
  </K-POINTS>


  <PARALLELISM>
       Paralellization over frequencies
       # of processes:     6
  </PARALLELISM>


  ======================================================================
  =  Frequency Loop                                                    =
  ======================================================================

  Computing E(     1 ) =    -10.00000 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.03 secs

  Computing E(    20 ) =     -9.71471 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.04 secs

  Computing E(    40 ) =     -9.41441 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.04 secs

  Computing E(    60 ) =     -9.11411 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.05 secs

  Computing E(    80 ) =     -8.81381 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.05 secs

  Computing E(   100 ) =     -8.51351 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.05 secs

  Computing E(   120 ) =     -8.21321 eV
  T matrix converged after avg. # of iterations    3.000

                              Total time spent up to now :     0.06 secs

  Computing E(   140 ) =     -7.91291 eV
  T matrix converged after avg. # of iterations    4.000

                              Total time spent up to now :     0.06 secs

  Computing E(   160 ) =     -7.61261 eV
  T matrix converged after avg. # of iterations    5.000

                              Total time spent up to now :     0.06 secs

  Computing E(   167 ) =     -7.50751 eV
  T matrix converged after avg. # of iterations   17.000

                              Total time spent up to now :     0.07 secs


  ======================================================================
  =  Writing data                                                      =
  ======================================================================


  conductance written on file:    cond_bulk1_w90.dat

  doscond written on file:    doscond_bulk1_w90.dat

  ======================================================================

   <global routines>
             clock number :    14

                conductor :     0.00s CPU 

           datafiles_init :     0.01s CPU
     wannier90_to_interna :     0.00s CPU
            smearing_init :     0.02s CPU
                   cft_1z :     0.01s CPU (       2 calls,   0.007 s avg)
         hamiltonian_init :     0.00s CPU
              read_matrix :     0.00s CPU (       2 calls,   0.001 s avg)
              fourier_par :     0.00s CPU (       4 calls,   0.000 s avg)
             do_conductor :     0.34s CPU
        hamiltonian_setup :     0.00s CPU (     167 calls,   0.000 s avg)
            transfer_mtrx :     0.02s CPU (     334 calls,   0.000 s avg)
                    green :     0.01s CPU (     334 calls,   0.000 s avg)
              gzero_maker :     0.00s CPU (     501 calls,   0.000 s avg)
            transmittance :     0.00s CPU (     167 calls,   0.000 s avg)


